vm-setup/library/generate_macs.py

```diff
@@ -24,19 +24,19 @@
 version_added: "2.0"
 short_description: Generate a list of Ethernet MAC addresses
 description:
-   - Generate a list of Ethernet MAC addresses suitable for baremetal testing.
+   - Generate a list of Ethernet MAC addresses suitable for external testing.
 '''
 
 MAX_NUM_MACS = math.trunc(0xff / 2)
 
 
-def generate_baremetal_macs(nodes, networks):
-    """Generate an Ethernet MAC address suitable for baremetal testing."""
+def generate_external_macs(nodes, networks):
+    """Generate an Ethernet MAC address suitable for external testing."""
     # NOTE(dprince): We generate our own bare metal MAC address's here
     # instead of relying on libvirt so that we can ensure the
     # locally administered bit is set low. (The libvirt default is
     # to set the 2nd MSB high.) This effectively allows our
-    # fake baremetal VMs to more accurately behave like real hardware
+    # fake external VMs to more accurately behave like real hardware
```

Comment:
`external VMs` does not really make sense

Reply:
done